TYPO3.Neos/Resources/Public/JavaScript/Content/Components/ContentDimensionSelector.html

```diff
@@ -22,7 +22,7 @@
 	</div>
 	{{/if}}
 
-	<div {{bindAttr class=":neos-modal-centered :neos-content-dimension-selector-initial-translation-dialog controller.showInitialTranslationDialog:neos-show:neos-hide"}}>
+	<div {{bindAttr class=":neos-modal-centered :neos-content-dimension-selector-initial-translation-dialog controller.showInitialTranslationDialog:neos-show-flex:neos-hide"}}>
```

Comment:
and actually this way of hiding an element is not really a good way to do it in Ember

instead it could be wrapped with `{{#if controller.showInitialTranslationDialog}} xxx {{/if}}`